src/seb/registered_models/e5_mistral.py

```diff
@@ -24,6 +24,63 @@ def batched(iterable: Iterable[T], n: int) -> Iterable[tuple[T, ...]]:
         yield batch
 
 
+def task_to_instruction(task: Task) -> str:
+    if task.task_type in ["STS"]:
+        return "Retrieve semantically similar text"
+    if task.task_type in ["Summarization"]:
+        return "Given a news summary, retrieve other semantically similar summaries"
+    if task.task_type in ["BitextMining"]:
+        task_name_to_instruct: dict[str, str] = {
+            "Bornholm Parallel": "Retrieve parallel sentences in Danish and Bornholmsk",
+            "Norwegian courts": "Retrieve parallel sentences in Norwegian Bokmål and Nynorsk",
+        }
+        default_instruction = "Retrieve parallel sentences."
+        return task_name_to_instruct.get(task.name, default_instruction)
+    if task.task_type in ["Classification"]:
+        task_name_to_instruct: dict[str, str] = {
+            "Angry Tweets": "Classify Danish tweets by sentiment. (positive, negative, neutral)",
+            "DKHate": "Classify Danish tweets based on offensiveness (offensive, not offensive)",
+            "Da Political Comments": "Classify Danish political comments for sentiment",
+            "DaLAJ": "Classify texts based on linguistic acceptability in Swedish",
+            "LCC": "Classify texts based on sentiment",
+            "Language Identification": "Classify texts based on language",
+            "Massive Intent": "Given a user utterance as query, find the user intents",
+            "Massive Scenario": "Given a user utterance as query, find the user scenarios",
+            "NoReC": "Classify Norwegian reviews by sentiment",
+            "SweReC": "Classify Swedish reviews by sentiment",
+            "Norwegian parliament": "Classify parliament speeches in Norwegian based on political affiliation",
+            "ScaLA": "Classify passages in Scandinavian Languages based on linguistic acceptability",
+        }
```

Comment:
Should we move this either to a dict outside the function or a json file? (prompt inside the code is always a bit hard to read)

Reply:
This is how Microsoft does it so I just copied them, but we can of course do it differently if need be. I prefer this, because the prompts live close to the code, and it's easy to understand why what happens without having to look a million places.

Reply:
Let us keep it as is then - will you delete the scores for the model before you merge (as they are now invalid).

Reply:
I will try to rebase or cherry-pick my way to it